tldw_Server_API/app/core/Sandbox/runners/docker_runner.py

```
@@ -37,14 +44,37 @@ def __init__(self) -> None:
+            parts = out.split()
+            for i, tok in enumerate(parts):
+                if tok.lower() == "version" and i + 1 < len(parts):
+                    return parts[i + 1].rstrip(",")
+            return out
+        except Exception:
+            return None
+
     # Track active containers per run for cancellation
     _active_lock = threading.RLock()
     _active_cid: dict[str, str] = {}
+    _egress_net: dict[str, Optional[str]] = {}
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)